### PR TITLE
update to bootstrap 5, simplify, add navbar

### PR DESCRIPTION
**Pull request by shuesken**
Hello! This PR updates the bootstrap version to 5 and replaces some of the custom javascript and functionality with bootstrap functionality that does the same thing. Specifically both the starting jumbotron gets automatically sized, the sidebar, and the navbar. That should make the code a bit easier to understand and add on to.

It also adds a mobile navbar and the various links that people wanted added (Add your house, github, slack, /projects), addressing some of the issues

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -151,1213 +200,1221 @@ <h3 class="hsc-text-center lead">START A COMMUNITY</h3>
...
 
+    <!-- google analytics -->
+    <script>(function (i, s, o, g, r, a, m) { i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () { (i[r].q = i[r].q || []).push(arguments) }, i[r].l = 1 * new Date(); a = s.createElement(o), m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m) })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga'); ga('create', 'UA-83416444-1', 'auto'); ga('send', 'pageview');</script>
+</body>
 
-        <!-- google analytics -->
-        <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-83416444-1', 'auto');ga('send', 'pageview');</script>
-    </body>
-</html>
+</html>
```
Comment (noahpicard) [inline index.html:1420]: please add a newline at end of file!

File: css/style.css [hunk 2]
```diff
@@ -295,6 +314,57 @@ p.small {
...
+    scroll-margin-top: 140px;
+}
+
+.hsc-banner-inner hr {
+    border-top: 2px solid #eee !important;
+    width: 80%;
+    opacity: 1;
+    margin-top: 40px;
+    margin-bottom: 40px;
 }
```
Comment (noahpicard) [inline css/style.css:370]: could you add a newline at end of file?

File: css/style.css [hunk 3]
```diff
@@ -29,7 +29,6 @@ h1, h2, h3, h4, h5, h6 {
...
 
 .img-thumbnail {
-    height: 100%;
```
Comment (noahpicard) [inline css/style.css:32]: interesting that this still works! cool

**Review by noahpicard — APPROVED** [review]
Overall this looks fantastic! Thanks for adding the add House Button.

From my local testing 

image

image

File: css/style.css (hunk 3 above)
Comment (shuesken) [reply css/style.css:32]: Yeah I guess something about how bootstrap does sizing changed. `height: 100%` doesn't keep the image ratio.

**Comment by stevenelleman** [comment]
Hmmm I think this broke the nav bar in certain cases -- the navbar selection appears to be offset so that it's off by one. @shuesken could you please fix this case?

**Pull request merged**